scapy/compat.py

```diff
@@ -9,13 +9,22 @@
 """
 
 from __future__ import absolute_import
+import abc
 import base64
 import binascii
 import gzip
 import struct
+import sys
 
 import scapy.modules.six as six
 
+
+# Abstract Base Classes - this makes Python < 3.4 act like >= 3.4
+if sys.version_info >= (3, 4):
+    ABC = abc.ABC
+else:
+    ABC = abc.ABCMeta('ABC', (), {})
```

Comment:
Could you then update `scapy/contrib/http2.py` to use this ?
Also, I'm guessing you took that from the same stackoverflow answer than I did :smile: feel free to link to that if that's the case